portfolio/src/main/webapp/script.js

```diff
@@ -70,3 +70,14 @@ function contact() {
 		</ul>`; 
 	console.log('Contact!'); 
 }
+
+function getComments() {
+	fetch('/data').then(response => response.json()).then((comments) => {
+		console.log(comments);
+		const bodyElement = document.body;
+		bodyElement.innerHTML += 'Here are the comments: \n';
```

Comment:
No action needed. I personally don't find it useful to add to the bottom of the body. I prefer having a dedicated div and add to that so i can control the looks of the page with html and not java server. but it's up to you.

Reply:
Yeah, I think I was just doing that for now to keep the non-serve-let related changes to a minimum while I went through the exercises :)